src/views/app/detail-panel/edit/edit-view.jsx

```diff
@@ -238,6 +243,10 @@ export default function EditView({ mailId, folderId, setHeader, toggleAppBoard }
 
 	const editorId = useMemo(() => activeMailId ?? generateId(), [activeMailId]);
 
+	useEffect(() => {
+		if (actionChanged) dispatch(closeAllEditors());
```

Comment:
consider this one too

Reply:
oh fuck ,let me re check everything again